seldonite/analyze.py

```diff
@@ -23,4 +26,37 @@ def keywords_over_time(self, keywords: List[str]):
 
             #df.groupby
 
-        return df
\ No newline at end of file
+            return df
+
+
+        def proportion_of_countries(self):
+            self.collector._check_args()
+
+            spark_builder = self.collector._get_spark_builder()
+
+            with spark_builder.start_session() as spark_manager:
+                df = self.collector._fetch(spark_manager)
+
+                df = df.withColumn('all_text', psql.functions.concat(df['headline'], psql.functions.lit(' '), df['short_description']))
```

Comment:
Right now the default columns I'm using for any dataframes passing through the library are 'title', 'text', 'publish_date' and 'url', so change 'headline' and 'short_description' here to 'title' and 'text respectively. Later on we can maybe add functionality for custom dataframe column names, but for now we'll have to keep it uniform for simplicity